packages/sui-dashboard/src/index.js

```diff
@@ -35,7 +48,14 @@ module.exports.stats = async ({repositories, root, dry}) => {
     acc[repo] = dirs
       .filter(dir => dir.includes(repo))
       .filter(dir => suiComponents.some(sui => dir.includes(sui)))
-      .map(dir => dir.replace(/^.+(?<comp>@[a-z|-]+\/[a-z|-]+$)/, '$<comp>'))
+      .map(dir => {
+        const pkg =
+          getVersions && JSON.parse(fs.readFileSync(`${dir}/package.json`))
```

Comment:
there is code you are repeating that could be moved into its own function

Reply:
thks

Reply:
done